### PR TITLE
Add backup config load

### PR DESCRIPTION
**Pull request by vascograce**
* When somebody messes up their sequencing of tags, where the config tag loads after the event tag
  - Then as a backup we still load the configuration and warn users by means of log to change their sequencing

### CODE REVIEW

File: template.js [hunk 1]
```diff
@@ -76,10 +83,46 @@ if (eventName === 'unknown') {
...
+  // Function to ensure BillyPix is defined and properly queues commands
+  const BillyPixBackup = createArgumentsQueue(billyFunctionName, billyFunctionName + '.queue');
+  
+  // Setup BillyPix with the current timestamp
+  setInWindow(billyFunctionName + '.t', getTimestamp(), false);
+
+  // Generate the script URL with cache busting
+  const secondsBuste = 30*1000;
+  const epochRounded = secondsBuste * Math.ceil(getTimestamp() / secondsBuste);
+  const scriptUrl = cdnEndpoint + '?t=' + epochRounded + '&v=0.2.0';
```
Comment (foadbillygrace) [inline template.js:109]: What the version does here?
Do we wanna make this a variable and re use it in other URLs ?

File: template.js [hunk 2]
```diff
@@ -76,10 +83,46 @@ if (eventName === 'unknown') {
...
+}
+
+
 // Sanity check: BillyPix needs to be available
 if (getType(BillyPix) === 'undefined') {
-  log(billyFunctionName + ' not available in window, make sure to first run the "Billy Grace - Web Configuration" tag');
-  return data.gtmOnFailure();
+    
+    // Make it clear they the user messed up
+    log(billyFunctionName + ' not available in window, make sure to first run the "Billy Grace - Web Configuration" tag. For now we are still loading it, but this can lead to performance issued');
```
Comment (foadbillygrace) [inline template.js:122]: Nice!
Does this msg goes to GTM debug console only?
There is a way to let BG or client knows without GTML Debug mode?

**Review by foadbillygrace — APPROVED** [review]
LGTM!

File: template.js (hunk 2 above)
Comment (vascograce) [reply template.js:122]: Yeah so this log will show up in the browser. Only unfortunately I could not find a way to make it a warning log with google's current APIs for web templates

File: template.js (hunk 1 above)
Comment (vascograce) [reply template.js:109]: Yeah so this is the way we normally load the bundle, but you are right the version we don't actually use. Only the `t` part is useful to buste the cache automatically as the url changes after that interval

**Pull request merged**